.gitignore

```diff
@@ -2,8 +2,7 @@
 /dist
 *.dol
 *.elf
-
-ipl.rom
+*.rom
```

Comment:
Also lukewarm about this one but I'll accept it. The need for a ROM dump will eventually go away when we have a BS1 replacement, and unless you're trying to build for Qoob Pro, the build system is smart enough to work without now.

All the build artifacts can be cleaned up from gitignore now, since Meson enforces out-of-tree builds, and it adds its own in the build directory.

Reply:
Also emoved in the last rebase.